input/intro-notes/StructureDefinition-ehealth-servicerequest-intro.md

```diff
@@ -22,16 +22,19 @@ The validity depends on the variant of `occurrence[x]` as follows:
 * `occurrenceTiming`: Shall either be ad-hoc or recurring as follows:
     * `occurrenceTiming` ad-hoc: the elements of `Timing.repeat` shall adhere to: 
         * `boundsPeriod`: shall be specified and shall contain at least `start` containing a date-time with at least hours and minutes (and hence timezone). If present, `end` shall contain a date-time with at least hours and minutes (and hence timezone) which shall be equal to or after `start`.
-        * `frequency`: shall be specified
+        * `frequency`: optional
         * `count`: is optional, required when `countMax` is not empty
         * `countMax`: is optional 
+        * `duration`: optional, required when `durationUnit` is specified, and it must be pos-integer
+        * `durationUnit`: required when `duration` is specified. Shall be one of s, min, h, d, wk, a.
     * `occurrenceTiming` recurring: the elements of `Timing.repeat` shall adhere to:
```

Comment:
The initial text for reccuring should include text explaining that either "dayOfWeek" or "period" must be assigned so it is clear what the difference is between ad-hoc and recurring.